modules/ROOT/pages/overview-apikit-for-rest.adoc

```diff
@@ -3,4 +3,25 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-TBD
\ No newline at end of file
+APIkit for REST is a powerful framework designed to simplify the development and integration of RESTful APIs. REST (Representational State Transfer) is an architectural style for building web services, and APIkit for REST provides a comprehensive set of tools and features to streamline REST API development while ensuring compliance with industry standards.
+
+APIkit for REST fully supports RAML (RESTful API Modeling Language) and OAS (OpenAPI Specification), which are widely adopted formats for describing RESTful APIs. By leveraging RAML or OAS, you can define your API contracts, enabling consistent design, enhanced collaboration, and clear documentation.
+
+Features:
+
+* *Automatic Validation*: APIkit for REST automatically validates incoming requests and outgoing responses against your API contracts defined in RAML or OAS. This ensures that your APIs adhere to the specified rules and constraints, enhancing the overall quality and reliability of your API implementation.
+
+* *Code Generation*: The framework provides code generation capabilities that help streamline the implementation of your RESTful APIs. APIkit for REST can generate implementation templates based on your API contracts, reducing the amount of manual coding required and accelerating the development process.
+
+* *Intelligent Routing*: With APIkit for REST, incoming requests are intelligently routed to the appropriate endpoints based on the defined API contracts. This routing mechanism simplifies request handling, improves performance, and ensures that requests are processed by the correct API resources.
```

Comment:
In addition to the above feature, I think this is an excellent explanation for APIkit in general, not only for REST. I would move it to the general APIkit overview (if not already there).

Reply:
Added to the APIkit overview. Thanks!